index.html

```diff
@@ -1,257 +1,296 @@
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <button>See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="about_me">
+      <div class="about">
+        <h1>About me</h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <button>Get My Resume</button>
+      </div>
+      <hr />
+
+        <ul class="skills">
+          <li class="skill" style="list-style-image: url(./assets/images/diamond.png)">
+          
+            <h1>Languages</h1>
+            <ul class="skill-list">
+              <li>
+                <h5>JavaScript</h5>
+              </li>
+              <li>
+                <h5>Ruby</h5>
+              </li>
+              <li>
+                <h5>Html</h5>
+              </li>
+              <li>
+                <h5>Css</h5>
+              </li>
+              <li>
+                <h5>Python</h5>
+              </li>
+            </ul>
+         
+          </li>
+          <li class="skill" style="list-style-image: url(./assets/images/rect.png)">
+          
+            <h1>Frameworks</h1>
+            <ul class="skill-list">
+              <li>
+                <h5>Bootstrap</h5>
+              </li>
+              <li>
+                <h5>Ruby on Rails</h5>
+              </li>
+              <li>
+                <h5>RSPec</h5>
+              </li>
+              <li>
+                <h5>CapyBara</h5>
+              </li>
+              <li>
+                <h5>Selenium</h5>
+              </li>
+            </ul>
+         
+          </li>
+          <li class="skill" style="list-style-image: url(./assets/images/circld.png)">
+          
+            <h1>Skills</h1>
+            <ul class="skill-list">
+              <li>
+                <h5>Codekit</h5>
+              </li>
+              <li>
+                <h5>Github</h5>
+              </li>
+              <li>
+                <h5>Codepen</h5>
+              </li>
+              <li>
+                <h5>Gitlab</h5>
+              </li>
+              <li>
+                <h5>Terminal</h5>
+              </li>
+            </ul>
+          </li>
+        </ul>
+    </section>
+
+    <footer class="contact">
+      <p>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </p>
+      <form action="https://formspree.io/f/xlevolbp" method="post" class="contact-form">
+        <input maxlength="30" name="user_name" type="text" placeholder="Full name" class="inputField" required />
+        <input type="email" name="user_email" placeholder="Email address" class="inputField" required />
```

Comment:
- [ ] You did a great job working on this section. However, please stick as much as possible to the Figma design. 🎨  For this section, there are some things missing from this section such as the font, the button aligned to the left, and the colors of the social media section. 🤓 🛠️ 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/199822556-84af735a-e949-41da-9266-f8eb6f3db662.png) | ![imageedit_11_3142877896](https://user-images.githubusercontent.com/77069011/199822432-9152a91b-5ba6-421a-b2e9-a33a55701278.gif) |